### ISSUE TITLE
[Bug]: pip won't install for some reason...

### ISSUE THREAD

**Issue by Hikaruuuunnnnn** [Derpitron/Discord-OTP-Forcer#102]
### Description

So, I lost my discord account after restarting my laptop. Since my backup codes aren't working, I've decided to take the brutal path of brute-forcing the dang thing. It's my only option in the meantime until support does something useful. When I followed the instructions (downloading python, reading the guide, blah blah blah), pip wont install and the setup batch file thinks I've installed pip when I didn't. Keep in mind that I have little to no knowledge in python, nor how the heck does the inside work. I mean sure, it's technically a macro that will open a chrome tab which will input as many codes as it can till it opens, but I just need some help here, please?
![image](https://github.com/Derpitron/Discord-OTP-Forcer/assets/157684348/ee8d65d4-cd7a-4598-a4df-51886f0e68f2)
Here is the screenshot of the problem, I blurred my real name for privacy reasons (I named some of the folders with my real name because I use a family laptop).

### Minimal Reproduction

TLDR, Pip didn't install, cfg.yml loaded and start batch file didn't open correctly

### Error

![image](https://github.com/Derpitron/Discord-OTP-Forcer/assets/157684348/d55f8bc9-5007-409d-a8f9-95ce2d6b8dd5)
:(

### Operating System Version

Windows 11

### Python Version

3.10

### Program Version 

idk

**Comment by Derpitron** [comment]
Not an issue with Discord-OTP-Forcer. Please don't clog the issue tracker with irrelevant/off topic requests.
